test/integration/test/stream.js

```diff
@@ -0,0 +1,83 @@
+const {AudioConfig} = require('../../../dist/src/models/streaming/AudioConfig');
+const {RevAiStreamingClient} = require('../../../dist/src/streaming-client');
+const configHelper = require('../src/config-helper');
+const fs = require('fs');
+const assert = require('assert');
+const colors = require('colors');
+
+(async (done) => {
```

Comment:
I dont think this `done` should be there, probably a relic from when we wanted to use jest